packages/components/src/components/hds/pagination/numbered/index.ts

```diff
@@ -327,4 +376,18 @@ export default class HdsPaginationNumberedIndexComponent extends Component {
       onPageSizeChange(newPageSize);
     }
   }
+
+  elliptizedPageArrayItemAsNumber = (
```

Comment:
Continuing from the above comment: this seems extra to have to do... shouldn't it always be a number?

Reply:
It's either a number or an ellipsis.